src/stores/Frame/FrameStore.ts

```diff
@@ -128,6 +128,7 @@ export class FrameStore {
     public readonly wcsInfoForTransformation: AST.FrameSet;
     public readonly wcsInfo3D: AST.FrameSet;
     public readonly validWcs: boolean;
+    public readonly wcsSystem: SystemType;
```

Comment:
How about renaming it as `defaultWcsSystem`?

Reply:
Yes, that will be clearer. Just changed.